app/decorators/alchemy/content_editor.rb

```diff
@@ -8,6 +8,20 @@ def to_partial_path
       "alchemy/essences/#{essence_partial_name}_editor"
     end
 
+    def css_classes
+      css_classes = [
+        'content_editor',
+        essence_partial_name,
+      ].compact
+    end
+
+    def dom_data_attributes
```

Comment:
I do not think we need `dom_` in the name as all of this is used in the DOM

```suggestion
    def data_attributes
```